CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 ### Added
 
 - Added `MeansImplicitUse` attribute to `RequireAttribute` to reduce warnings in Rider IDE. [#1462](https://github.com/spatialos/gdk-for-unity/pull/1462)
+- Added tooltips to the GDK tools configuration options. [#1470](https://github.com/spatialos/gdk-for-unity/pull/1470)
```

Comment:
Since the window is under Project Settings > Spatial OS, should the changelog use that name instead?
```suggestion
- Added tooltips to the SpatialOS Project Settings. [#1470](https://github.com/spatialos/gdk-for-unity/pull/1470)
```

Reply:
I would prefer this name, but we'd need to rename the `GdkToolsConfiguration` class and related things too

Reply:
The changelog is for end users though, and they don't know/care about the underlying implementation names. But they do see this in the `Project Settings > SpatialOS` window.

Reply:
In the make your own section we tell users to create a gdk tools config file. I was thinking it'd be strange to tell users to create a "SpatialOS project settings" file called `GdkToolsConfiguration.json`

Reply:
actually that doesn't really matter too much, ignore me

Reply:
That's a valid issue that we should probably resolve in [UTY-2692](https://improbableio.atlassian.net/browse/UTY-2692), but I don't think its a valid argument for not using the new name for new public facing docs. 

I.e. - we would never write 'open the GDK Tools Configuration window' in new docs after we've moved it.

Reply:
yup exactly^

Reply:
Race condition in replying to comments 😂